src/tzdata/build.jl

```diff
@@ -68,7 +79,11 @@ function build(version::AbstractString="latest", regions=REGIONS)
         rm(joinpath(COMPILED_DIR, file), recursive=true)
     end
 
-    version = build(version, regions, ARCHIVE_DIR, TZ_SOURCE_DIR, COMPILED_DIR, verbose=true)
+    version = build(
```

Comment:
Seemed more readable the other way, IMO.

Reply:
Was modified to fit within the 92 column line length